hack/managed-kyma-migration/readme.md

```diff
@@ -103,10 +67,37 @@ adds the Istio module to the Kyma CR without overwriting existing module configu
 
 Perform the rollout to Stage together with the SRE team. Since they have already performed the rollout for other modules, they might suggest a different rollout strategy.
 
-1. Apply the module template to Stage Control Plane.
-2. `kcp login` to Stage, select some SKRs on `Kyma-Test/Kyma-Integration`, and run `migrate.sh` on them using `kcp taskrun`.
-3. Verify if the migration was successful on the SKRs by checking the status of Istio CR and the reconciler's components.
-4. Run `migrate.sh` for all SKRs in Kyma-Test and Kyma-Integration global accounts.
-5. Verify if the migration worked as expected.
-6. Run `migrate.sh` for the whole Canary landscape.
-7. Verify if the migration worked as expected.
\ No newline at end of file
+#### Prerequisites
+
+- Reconciliation is disabled for the Stage environment. See PR #4486 to the `kyma/management-plane-config` repository.
+
+#### Migration procedure
+
```

Comment:
1. Push the module to experimental `channel` in `kyma/module-manifests` repository.
2. Test that experimental `channel`deploys as expected by manually enabling it on a Stage managed cluster